### ISSUE TITLE
How to solve short trading with ta4j?

### ISSUE THREAD

**Issue by jansoren** [ta4j/ta4j#318]
Do you have any thoughts on how to implement short trading? Should we implement new indicators that is only for shorting, or should we expand on the already implemented indicators with the ability to short aswell?

**Comment by team172011** [comment]
I don't understand you. You can initialize Strategies with OrderTyp.SHORT to execute sell-buy trades if you run it

**Comment by jansoren** [comment]
So the OrderType.SELL means SHORT? https://github.com/ta4j/ta4j/blob/ea5182a62cc71297b7f646290fdeca39c04c490a/ta4j-core/src/main/java/org/ta4j/core/Order.java#L61

Then this should be refactored from BUY to LONG and from SELL to SHORT.

**Comment by team172011** [comment]
Yes, it describes if it is a selling or a buying order. And in construct of Strategy you describe with this parameter the kind of entry order for each trade. But sell is not always equal to going short. You can *sell* securities you own and the possibility to sell securities you don't own means going short. Maybe renaming the Startegy parameter makes sense, but not the order type, because SELL describes the type in its generally meaning (sell or short.. )

**Comment by team172011** [comment]
I am unsure at the moment, but this question has been asked several times before, so I will add a question-answer to the FAQ section of the wiki
